models/platform-limits/src/cart-discounts/builder.spec.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TCartDiscountLimitsProjection } from './types';
+import * as CartDiscountsPlatformLimits from './index';
+
+describe('building', () => {
+  it(
+    ...createBuilderSpec<
+      TCartDiscountLimitsProjection,
+      TCartDiscountLimitsProjection
+    >(
+      'default',
+      CartDiscountsPlatformLimits.random(),
```

Comment:
Since this will just create a model instance with a null property, perhaps we can use a its `with-limit-and-current` preset instead so we test the value generated is correct.

Reply:
Done https://github.com/commercetools/test-data/pull/656/commits/3c8d9ad3478dd81b08fae8e146a8c73405b42e3f